Courses/Views/Screens/HomeView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  HomeView.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 07.10.2022.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+    @State var hasScrolled = false
+    
+    var body: some View {
+        ZStack {
+            Color("Background").ignoresSafeArea()
+            
+            ScrollView {
+                scrollDetection
+                featured
+                Color.clear.frame(height: 1000)
+            }
+            .coordinateSpace(name: "scroll")
```

Comment:
@DCHRKV for what `.coordinateSpace(name: "scroll")` is used?

Reply:
@Den-Ree , it's used to detect the scroll position in order to animate Navigation Bar

Reply:
got it, thanks. If you use value `scroll` somewhere else for detection, then you need to move it in constant and don't use hardcoded values

Reply:
@Den-Ree , got it, thx